sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/SoftDeleteState.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.RecoveryServices.Models
+{
+    /// <summary> The SoftDeleteState. </summary>
+    public readonly partial struct SoftDeleteState : IEquatable<SoftDeleteState>
```

Comment:
The naming is too general, add it to `prepend-rp-prefix` in [autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/autorest.md)